walkgen_params:
  planeseg: False # Use URDF of the environment or planeseg visualisation.

  world:
    path: ""
    urdf: "data/urdf/lab_scene.urdf" # Env URDF path
    heightmap: "data/lab_scene.dat"  # Heightmap path

  params:
    margin: 0.06 # inner surface margin
    n_points: 6 # Maximum number of vertices for the surfaces
    method_id: 3 # Method to remove overlapping
    poly_size: 10 # Number of maximum point for convex decomposition.
    min_area: 0.03 # Minimum area to keep the surfaces
    N_phase: 6 # Number of step to proceed (--> N_phase * n_gait step in SL1M)
    N_phase_return: 2 # Number of step to return (N_phase_return surfaces for each foot)
    com: False # Optimisation of the CoM
    multiprocessing_mimic: True # Mimic the behaviour of multiprocessing by using the surfaces of the previous optimisation.

  gait:
    type: "Walk" # Only "Walk" or "Trot" working
    dt: 0.01
    N_ss: 20 # 30 for Trot
    N_ds: 5 # 0 for Trot

  trajectory: # Degree of the polynomial curves
    nx: 5
    ny: 5
    nz: 6
